onnxruntime/core/providers/openvino/backends/basic_backend.h

```diff
@@ -69,18 +74,18 @@ class InferRequestsQueue {
   void printstatus() {
     std::cout << "printing elements of the vector (infer_requests_): " << std::endl;
     for (auto i = infer_requests_.begin(); i != infer_requests_.end(); ++i) {
-      std::cout << *i << " ";
+      i->get()->query_status();
```

Comment:
with 2022.1 and current changes, it would just print up something like this:

**printing elements of the vector (infer_requests_):** 
ovInfReq.query_state() ovInfReq.query_state() ovInfReq.query_state() ovInfReq.query_state() ovInfReq.query_state() ovInfReq.query_state() ovInfReq.query_state() ovInfReq.query_state() 

Make required changes in ov_infer_request::query_status() method in the file ov_interface.cc